data/mods/Generic_Guns/gg_ammo_override.json

```diff
@@ -426,4 +426,4 @@
         "count" : 10,
         "effects" : ["INCENDIARY"]
     }
-]
```

Comment:
> The program you used for line counting does apparently not care about this.
Vim for my copy, going to https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/mods/Generic_Guns/gg_ammo_override.json in my browsers (Firefox and Iron) for the master.